CMakeLists.txt

```diff
@@ -179,8 +179,6 @@ configure_file(config.h.in config.h)
 if(WIN32)
     install(FILES COPYING DESTINATION .)
     install(FILES README.md DESTINATION .)
-    install(FILES betagrph.wad DESTINATION .)
```

Comment:
How about `watermap.wad`, is this used anywhere? I just added it back then because it was also in the archive.

Reply:
From boom.txt:
`WATERMAP.WAD......File that allows DCK to insert WATERMAP texture`
We also have DCK configs in `/configs`.
I doubt anyone is using DCK.

Reply:
Yes, right, we can remove that a well.